src/qt/addressbookpage.cpp

```diff
@@ -107,7 +107,7 @@ AddressBookPage::AddressBookPage(const PlatformStyle *platformStyle, Mode _mode,
         ui->newAddress->setVisible(true);
         break;
     case ReceivingTab:
-        ui->labelExplanation->setText(tr("These are your Bitcoin addresses for receiving payments. It is recommended to use a new receiving address for each transaction."));
+        ui->labelExplanation->setText(tr("These are your Bitcoin addresses for receiving payments. Use the request payment button in the receive tab to create new addresses."));
```

Comment:
If the button is getting renamed, it doesn't make sense to refer to the old name here.

Reply:
Agreed